### PR TITLE
FEA-4627: fedx_codeowners_file

### PR DESCRIPTION
**Pull request by sourcegraph-wk**
# [FEA-4627](https://jira.atl.workiva.net/browse/FEA-4627)
Issue Status

Adds a CODEOWNERS file to all of the repos officially owned by the fedx
Repo ownership was decided by [`repo:has.meta(team:fedx)`](https://workiva.sourcegraphcloud.com/search?q=repo%3Ahas.meta%28team%3Afedx%29) so therefore transitively [CID](https://cid.workiva.net/teams)

[_Created by Sourcegraph batch change `Workiva/fedx_codeowners_file`._](https://workiva.sourcegraphcloud.com/organizations/Workiva/batch-changes/fedx_codeowners_file)

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+    groups:
+      gha:
+        patterns: ["*"]
+  - package-ecosystem: pub
+    versioning-strategy: increase
```
Comment (robbecker-wf) [inline .github/dependabot.yml:11]: Not sure if we'll need to avoid major version raises or not?

**Review by robbecker-wf — APPROVED** [review]
Good as-is. We can always review or change dependabot major version bump strategy if it becomes a problem.

**Comment by matthewnitschke-wk** [comment]
QA +1

- [x]  CI does indeed pass, changes just to CI

🚀 @Workiva/release-management-p 🚢

**Comment by matthewnitschke-wk** [comment]
🚀 @Workiva/release-management-p 🚢

**Review by dustinlessard-wf — APPROVED** [review]

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**